### PR TITLE
this commit adds solution to ticket 5 view on individual page

### PR DESCRIPTION
**Pull request by hja7333**
I forgot to create a new branch this morning, checked this with Joe, he ok'd this pr on the same branch. Sorry! Also I have not been able to delete the bullet points on the home page, I thought I could do this on the next branch. I hope this is ok.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,13 +4,15 @@ import "./App.css";
...
 import Article from "./components/Article";
 import NavBar from "./components/NavBar";
+import SingleArticle from "./pages/SingleArticle";
 function App() {
   return (
     <>
       <NavBar />
       <Routes>
         <Route path="/" element={<HomePage />} />
         <Route path="/article" element={<Article />} />
```
Comment (JVal-NC) [inline src/App.js]: Are the routes right for the plural / singular of article?

There's something odd going on by not being able to navigate to /article without going to the entry point of the page. Can this be fixed please.

File: src/pages/SingleArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        setArticle(data.article);
+        setIsLoading(false);
+      });
+  }, [article_id]);
+
+  return isLoading ? (
+    <p>Loading article</p>
+  ) : (
+    <main>
+      {/* <img src={article.img} alt={article.name} /> */}
```
Comment (JVal-NC) [inline src/pages/SingleArticle.jsx]: Please remove this.

File: src/components/Article.jsx [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
-      <h3 className={styles.title}>{title}</h3>
-      <img
-        className={styles.article_img}
-        src={article_img_url}
-        alt={`image of ${title}`}
-      ></img>
-      <p>Author {data.author}</p>
-      <p>posted at {data.created_at}</p>
-    </article>
+    <Link to={`/articles/${article_id}`}>
```
Comment (JVal-NC) [inline src/components/Article.jsx:8]: Can this be styled not to show a bullet point

**Review by JVal-NC — COMMENTED** [review]
Looks good but some work required on navigation so URLs dont only work by clicking links.

**Review by Rayhaan2k9 — APPROVED** [review]
Have approved for you Al but please do remove the bullet points in styling 🙂

**Pull request merged**